src/pipeline.jl

```diff
@@ -274,12 +275,35 @@ function _subplot_setup(plt::Plot, plotattributes::AKW, kw_list::Vector{KW})
 
     # do we need to link any axes together?
     link_axes!(plt.layout, plt[:link])
+    return nothing
 end
 
 function series_idx(kw_list::AVec{KW}, kw::AKW)
     Int(kw[:series_plotindex]) - Int(kw_list[1][:series_plotindex]) + 1
 end
 
+function _add_plot_title!(plt)
+    plot_title = plt[:plot_title]
+    if plot_title != ""
+        the_layout = plt.layout
+        vspan = plt[:plot_titlevspan]
+        plt.layout = grid(2, 1, heights=(vspan, 1 - vspan))
+        plt.layout.grid[1, 1] = subplot = Subplot(plt.backend, parent=plt.layout[1, 1])
+        plt.layout.grid[2, 1] = the_layout
+        subplot.plt = plt
+        # propagate arguments plt[:plot_titleXXX] --> subplot[:titleXXX]
+        for sym ∈ filter(x -> startswith(string(x), "plot_title"), keys(_plot_defaults))
+            subplot[Symbol(string(sym)[length("plot_") + 1:end])] = plt[sym]
+        end
+        plt[:force_minpad] = nothing, 0px, nothing, 0px
```

Comment:
why is that?

Reply:
This is for getting the output images at https://github.com/JuliaPlots/Plots.jl/pull/3608#issue-683048141, changing color, ...

If you remove these lines, you parametrize the `plot_title` with `plot_titlefontcolor`, `plot_titlelocation` does not work, does it ?.

Reply:
No, I was referring to the `force_minpad`

Reply:
And the `force_minpad` thing is to avoid too much vertical white space  https://github.com/JuliaPlots/Plots.jl/pull/2690#issuecomment-873423600. It's a hack, but I haven't found any other solution.

The unwanted white space is induced by `_update_min_padding!`: https://github.com/JuliaPlots/Plots.jl/blob/c97900f7ee641ce754577b731a8f13e373a8ae73/src/layouts.jl#L282 which is recursive.